### ISSUE TITLE
Only hooks work on live.

### ISSUE THREAD

**Issue by Geestig** [barbajs/barba#453]
Hi there, 

First and foremost thanks for the great work you did with Barba so far! However upon trying to implement a simple site with it I've come across an issue I can't seem to fix.

See my basic code:

`

    barba.hooks.beforeEnter((data) => {
      const currentHtml = new DOMParser().parseFromString(data.next.html, 'text/html');
      const newHead = currentHtml.querySelector('head style');
      const oldHead = document.querySelector('head style');
      if (newHead) {
        if (oldHead) {
          oldHead.innerHTML = newHead.innerHTML;
        } else {
          document.querySelector('head').appendChild(newHead);
        }
      }

      this.initScripts();
    });

    barba.hooks.afterLeave(() => {
      window.scrollTo(0, 0);
    });

    barba.init({
      timeout: 5000, // default is 2000ms
      debug: true,
      transitions: [{
        leave() {
          // do something with `current.container` for your leave transition
          // then return a promise or use `this.async()`
          console.log('LEAVE');
        },
        enter() {
          // do something with `next.container` for your enter transition
          // then return a promise or use `this.async()`
          console.log('ENTER');
        }
      }]
    });
`

Locally this works like a charm. 

However, upon deploying it only the hooks seem to work (beforeEnter, afterLeave) but the LEAVE and ENTER logs are never triggered. I've also tried adding this.async() but this also broke the navigation on my local version.

TLDR: When deployed navigation happens as if there is no barba initialized (but hooks are triggered) while locally the navigation works as intended.

P.S.: The invite link for Slack is no longer active.

EDIT: When I place the init before the hooks, the hooks aren't triggered either on live but still are on local. However - the beforeEnter doesn't get triggered until actually navigating while it does trigger 1 time when it's initialized before the init.

EDIT EDIT: Downgrading to 2.3.15 did not help either. Hoped #444 might have been something similar.

EDIT EDIT EDIT: It only seems to work when the url is absolute like '/pagename' and not when it's the full URL starting with http or https.
